examples/podsecuritypolicy/rbac/policies.yaml

```diff
@@ -23,7 +23,7 @@ spec:
   - min: 1
     max: 65536
 ---
-apiVersion: extensions/v1beta1
+apiVersion: policy/v1beta1
```

Comment:
I was going to suggest updating the addons too, but then I realized that could result in duplication of all of the policies after an upgrade...

Reply:
> was going to suggest updating the addons too, but then I realized that could result in duplication of all of the policies after an upgrade...

it shouldn't... the old and new API group are colocated... existing extensions/v1beta1 objects would also surface under policy/v1beta1

Reply:
oh, great. In that case, @php-coder can you update the addons too?